### PR TITLE
WorkspaceView in jupyter: ask before delete?

### LINKED ISSUE #424: WorkspaceView in jupyter: ask before delete?
right now pressing the trash can button immediately deletes the recording, but maybe it should ask the user if they really want to do it first since this cannot be undone easily

### PR DESCRIPTION
**Pull request by magland**
Fixes #424

Add confirm delete and clean up code in the NiceTable, SortingsTable, and RecordingsTable

### CODE REVIEW

**Comment by magland** [comment]
image

File: src/extensions/common/NiceTable.tsx [hunk 1]
```diff
@@ -109,6 +142,51 @@ const NiceTable: FunctionComponent<Props> = ({
...
+        <span>
+            Confirm delete?
+            <IconButton
+                title={title}
+                onClick={handleClick}
+            ><Delete /></IconButton>
+            <IconButton
+                title={"Cancel"}
+                onClick={handleCancel}
+            >🗙</IconButton>
```
Comment (jsoules) [inline src/extensions/common/NiceTable.tsx:174]: This is a cancel emoji. For some reason, it's not displaying correctly with my installed font set, either on github or in vscode or (rather more importantly) through the web ui. To ensure consistency across users' font sets, we should probably encode it with a `FontAwesome`/`Material`/etc embedded font icon as we do elsewhere.

**Review by jsoules — CHANGES_REQUESTED** [review]
Changes make sense and work for me in the UI, but I've left a note about the cancel icon not displaying correctly--looks like there's a raw emoji in the source code, which isn't playing nicely with my installed font set.

File: src/extensions/common/NiceTable.tsx (hunk 1 above)
Comment (magland) [reply src/extensions/common/NiceTable.tsx]: I have replaced it by `&#10006;` which should do the trick.

Comment (jsoules) [reply src/extensions/common/NiceTable.tsx]: Confirming it looks good now.

**Pull request merged**